sample-projects/java-single-project/test.init.gradle

```diff
@@ -0,0 +1,20 @@
+initscript {
```

Comment:
I don't quite get what this is doing, or what makes the `java-single-project` sample project different to require this. Same for the sibling `test-local` script.

Reply:
Well caught: those files were added in error. Fixed c59c4f2